manifests/bootstrap_rpm.pp

```diff
@@ -20,7 +20,7 @@
   $katello_rhsm_setup_script = 'katello-rhsm-consumer'
   $katello_rhsm_setup_script_location = "/usr/bin/${katello_rhsm_setup_script}"
 
-  $candlepin_consumer_name = "katello-ca-consumer-${rhsm_hostname}"
+  $bootstrap_rpm_name = "katello-ca-consumer-${rhsm_hostname}"
   $candlepin_consumer_summary = "Subscription-manager consumer certificate for Katello instance ${rhsm_hostname}"
   $candlepin_consumer_description = 'Consumer certificate and post installation script that configures rhsm.'
```

Comment:
These appear to be unused now. I do wonder if it doesn't make sense to keep those in.